Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/NavigationLinkWidget.cpp

```diff
@@ -41,6 +41,7 @@ namespace EMStudio
 
         connect(&m_plugin->GetAnimGraphModel(), &AnimGraphModel::FocusChanged, this, &NavigationLinkWidget::OnFocusChanged);
         connect(&m_plugin->GetAnimGraphModel(), &AnimGraphModel::dataChanged, this, &NavigationLinkWidget::OnDataChanged);
+        connect(&m_plugin->GetAnimGraphModel(), &AnimGraphModel::RemoveAll, this, &NavigationLinkWidget::ClearModelIndex);
```

Comment:
I think you could subscribed to the beginRemoveRows signal and clear out the m_modelIndexes one by one, then update the breadCrumbs.